File: lista_01/01/SomaDeNumeros.py
def solicitar_numeros_e_somar():
    soma = 0
    total_numeros = 10
    print(f"Por favor, insira {total_numeros} números inteiros.")

    for i in range(1, total_numeros + 1):
        while True:
            try:
                numero = int(input(f"Digite o número {i}: "))
                break
            except ValueError:
                print("Por favor, insira um valor inteiro válido.")
        
        soma += numero

    print("A soma dos números é:", soma)

if __name__ == "__main__":
    solicitar_numeros_e_somar()

File: lista_01/01/SomaDeNumeros.java
import java.util.Scanner;

public class SomaDeNumeros {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int soma = 0;
        int totalNumeros = 10;

        System.out.println("Digite " + totalNumeros + " números inteiros:");

        for (int i = 1; i <= totalNumeros; i++) {
            System.out.print("Número " + i + ": ");
            int numero = scanner.nextInt();
            soma += numero;
        }

        System.out.println("A soma dos números é: " + soma);
        scanner.close();
    }
}
